02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -11,34 +11,53 @@ output:
 
 # Purpose of this analysis
 
-This example is one of pathway analysis module set, we recommend looking at the [pathway analysis introduction](https://alexslemonade.github.io/refinebio-examples/02-microarray/pathway-analysis_microarray_00_intro.html) to help you determine which pathway analysis method is best suited for your purposes.
+This example is one of pathway analysis module set, we recommend looking at the [pathway analysis table below](#how-to-choose-a-pathway-analysis) to help you determine which pathway analysis method is best suited for your purposes.
 
 This particular example analysis shows how you can use over-representation analysis (ORA) to determine if a set of genes (e.g., those differentially expressed using some cutoff) shares more or fewer genes with gene sets/pathways than we would expect at random.
 This pathway analysis method does not require any particular sample size, since the only input from your dataset is a set of genes of interest [@Yaari2013].
 
 ⬇️ [**Jump to the analysis code**](#analysis) ⬇️
 
+### What is pathway analysis?
+
+We refer to any technique that uses predetermined sets of genes that are related or coordinated in their expression in some way (e.g., participate in the same molecular process, are regulated by the same transcription factor) to interpret a high-throughput experiment as pathway analysis.
+In the context of refine.bio, we use these techniques to analyze and interpret genome-wide gene expression experiments.
+The rationale for performing pathway analysis is that looking at the pathway-level may be more biologically meaningful than considering individual genes, especially if a large number of genes are differentially expressed between conditions of interest.
+In addition, many relatively small changes in the expression values of genes in the same pathway could lead to a phenotypic outcome and these small changes may go undetected in differential gene expression analysis.
+
+We highly recommend taking a look at [Ten Years of Pathway Analysis: Current Approaches and Outstanding Challenges](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1002375) from @Khatri2012 for a more comprehensive overview and reading the primary publications and documentation of the methods and sources we will introduce below.
+
+### How to choose a pathway analysis?
+
```

Comment:
The spacing and the format of the below table makes it a bit difficult to read in the rendered html file i.e:
<img width="852" alt="Screen Shot 2020-11-16 at 4 46 12 PM" src="https://user-images.githubusercontent.com/43576623/99311912-4b7b2b00-282b-11eb-98ed-0302b03f99a3.png">

Is there a way that we can have the text aligned under the appropriate column headers? It may have to do with the spacing between each section of text?

Reply:
I was editing this in a markdown file and silly me, I assumed it should also be fine in Rmd. Not so. Good catch.

Reply:
Rmd doesn't like me having bulleted lists inside my table, while markdown is fine with it. Hmm....

Reply:
So apparently Rmarkdown has problem with making a list in a markdown table with `<ul><li>` (regular markdown is fine with this) but Rmarkdown is okay with `<br>` in a table so that's what I switched to here.